### PR TITLE
Ejercicios Semana 2

### PR DESCRIPTION
**Pull request by sheifg**
### Resolution of class 3 exercise
- I have checked the outline of the file `index.html` using (https://validator.w3.org/nu/#textarea)
- I have changed this file including the semantic tags such as `article`, `section`, `header` and `footer`
- After that I have made *fork* of the week 2 repository and then cloned it to my HOME using command `git clone "link"`
- I have created a folder with my discord name and included the `index.html`, `style.css` and images folder
- Then I have used the following command: `git add *`, `git commit -m "Including semantic elements in HTML"` and`git push`
- To conclude I have performed my PR

### CODE REVIEW

File: student-cvs/Sheila_FG-7067/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                </section>
+            </article>
+            <section>
+                <p class="parrafo">For any inquires, don't hesitate to <a href="contact.html">contact me</a></p>
+            </section>
+            <br>
+            <footer>
+                <div class="link">
+                    <img class="github-mark" src="./Images/github-mark.png" alt="GitHub mark">
+                    <a href="https://github.com/sheifg">Mi perfil de GitHub</a>
```
Comment (diirf) [inline student-cvs/Sheila_FG-7067/index.html:90]: You can use `target="_blank"` for open your GitHub page in a new tab

**Review by diirf — APPROVED** [review]
Hi! I liked your page, I just left you a few recommendations for the future.

File: student-cvs/Sheila_FG-7067/contact.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <label for="name">Name</label>
+                <input id="name" name="name" type="text" placeholder="Enter your full name" required>
+            </div>
+            <div class="form-group">
+                <label for="email">Email</label>
+                <input id="email" name="email" type="email" placeholder="Enter your email" required>
+            </div>
+            <div class="form-group">
+                <label for="phone">Phone</label>
+                <input id="phone" name="phone" type="tel" placeholder="Enter your phone number" required>
```
[the remark this answers is not in the document]
Comment (sheifg) [reply student-cvs/Sheila_FG-7067/contact.html:24]: I didn't know about this, I will definitely use it in the future :)

File: student-cvs/Sheila_FG-7067/index.html (hunk 1 above)
Comment (sheifg) [reply student-cvs/Sheila_FG-7067/index.html:90]: Thanks for the recommendation, I think you are right and it would be better to open it in a new page :)

**Review by JoeOsG — APPROVED** [review]
### Bien.

**Comment by sheifg** [comment]
Gracias @JoeOsG :)

**Pull request merged**